### DISCUSSION TITLE
PR ?

### DISCUSSION THREAD

**Discussion by DHIMANvivek** [Computer-Science-and-Engineering-GNDEC/Hackathon-1.1#126] [Q&A]
How to add PR
Please Any One Elaborate

**Comment by PriyankaJhamb** [comment]
We can send pull request by clicking the option of Pull Request. 
![image](https://user-images.githubusercontent.com/74251229/110921460-92de7780-8344-11eb-9994-1350a496d6df.png)

**Reply by DHIMANvivek** [reply]
Thankyou

**Reply by PriyankaJhamb** [reply]
Your Welcome
